src/utils/dom-event-utils.ts

```diff
@@ -12,67 +12,71 @@ const ARROW_UP_KEY_CODE = 38;
 const ARROW_RIGHT_KEY_CODE = 39;
 const ARROW_DOWN_KEY_CODE = 40;
 
-const validateEvent = (event, keyCode, key) => {
+function validateEvent(
+  event: KeyboardEvent,
+  keyCode: KeyboardEvent["keyCode"] | KeyboardEvent["which"],
+  key?: KeyboardEvent["key"]
+) {
   if (!event) return false;
   return event.keyCode === keyCode || event.which === keyCode || (key && event.key === key);
-};
+}
 
-export const isEnterEvent = event => {
+export function isEnterEvent(event: KeyboardEvent) {
```

Comment:
not that it matters to me that much, but why did you change from `const` to functions?

Reply:
@laviomri i'm not a huge favor of declaring `consts` when functions are needed - less evaluation, but i think it is clearer with TS (IMO)

Reply:
@orrgottlieb I'm not opinionated to consts or functions, but if we prefer having one over the other I think we should have a linting rule for that. Otherwise we'll have some PRs moving consts to functions and other PRs moving them back. 
what do you think?